storybook/stories/index.js

```diff
@@ -361,3 +368,15 @@ storiesOf('Screens/Login', module)
   .add('Start Screen', () => (
     <ScreenStart />
   ))
+  
+
+storiesOf('Screens', module)
```

Comment:
This block should be merged with code at lines 360-370.
Chapter 'Screens/Login' should be used.